### PR TITLE
feat: Add dedicated labels to agent status and OS

### PR DESCRIPTION
**Pull request by f0ssel**
Before:
image

After:
image

### CODE REVIEW

File: site/src/components/Resources/Resources.tsx [hunk 1]
```diff
@@ -103,40 +100,43 @@ export const Resources: FC<React.PropsWithChildren<ResourcesProps>> = ({
...
-                                  appName={app.name}
-                                  userName={workspace.owner_name}
-                                  workspaceName={workspace.name}
-                                  agentName={agent.name}
-                                />
-                              ))}
-                            </div>
-                          )}
-                        </>
+                          <div className={styles.accessLinks}>
```
Comment (f0ssel) [inline site/src/components/Resources/Resources.tsx:118]: I did this to try to get this column to stay the same width even if this condition isn't true. I'm not sure if it really worked.

**Comment by kylecarbs** [comment]
Tagged @BrunoQuaresma for review because him and I chatted about design work yesterday!

**Comment by BrunoQuaresma** [comment]
I like that! 👍

File: site/src/components/Resources/Resources.tsx [hunk 2]
```diff
@@ -103,40 +100,43 @@ export const Resources: FC<React.PropsWithChildren<ResourcesProps>> = ({
...
-                        {agent.name}
-                        <div className={styles.agentInfo}>
-                          <span className={styles.operatingSystem}>{agent.operating_system}</span>
-                          {WorkspaceStateEnum[workspaceStatus] !==
-                            WorkspaceStateEnum["stopped"] && (
-                            <span style={{ color: agentStatus.color }} className={styles.status}>
+                      <TableCellDataPrimary highlight>{agent.name}</TableCellDataPrimary>
+                      <div className={styles.data}>
+                        <div className={styles.dataRow}>
+                          <strong>status: </strong>
```
Comment (BrunoQuaresma) [inline site/src/components/Resources/Resources.tsx:106]: Just a convention on FE. We have a `Language` object that should be used to store texts in the UI.

File: site/src/components/Resources/Resources.tsx [hunk 3]
```diff
@@ -204,4 +204,23 @@ const useStyles = makeStyles((theme) => ({
   status: {
     whiteSpace: "nowrap",
   },
+
+  data: {
```
Comment (BrunoQuaresma) [inline site/src/components/Resources/Resources.tsx:208]: I would apply the same styles from the workspace cell - how it is doing with the metadata info - but if it is already applying that, I'm good!

**Review by BrunoQuaresma — APPROVED** [review]
Just minor things that are not blockers.

File: site/src/components/Resources/Resources.tsx (hunk 3 above)
Comment (f0ssel) [reply site/src/components/Resources/Resources.tsx:202]: This is taken straight from that component 👍

**Review by BrunoQuaresma — APPROVED** [review]

**Pull request merged**